src/components/home/elements/Record.vue

```diff
@@ -233,6 +251,14 @@ export default {
                 } else {
                     return 'Leased Out Canceled'
                 }
+            } else if (this.txRecord['type'] === CONTRACT_CREATE_TX) {
+                return 'Register Contract'
+            } else if (this.txRecord['type'] === CONTRACT_EXEC_TX) {
+                if (this.txRecord.status === 'Success') {
+                    return 'Execution Contract Success'
+                } else {
+                    return 'Execution Contract Fail'
+                }
```

Comment:
why register contract didn't check the status here and add extra check in line 31-38